lib/chewy/journal/apply.rb

```diff
@@ -1,6 +1,8 @@
 module Chewy
   class Journal
     module Apply
+    module_function
```

Comment:
also, this makes every function in this module a module function, is this intended? Previously only `:since` was a module function